### PR TITLE
Recurrence UI step 5 : Tweaks and fixes

### PR DESCRIPTION
**Pull request by ptbrowne**
Lots of little things here. Commits are self explaining and/or
have descriptions.

### CODE REVIEW

File: src/components/Header/__snapshots__/Header.spec.jsx.snap [hunk 1]
```diff
@@ -1,41 +1,96 @@
...
+    >
+      content
+    </div>
+  </div>,
+  <div
+    className="HeaderColor_normal"
+  >
+    content
+  </div>,
+]
```
Comment (Crash--) [inline src/components/Header/__snapshots__/Header.spec.jsx.snap:19]: I don't understand why we have an array with twice the content here?

File: src/ducks/recurrence/RecurrencePage.jsx [hunk 2]
```diff
@@ -54,6 +55,9 @@ const useDocument = (doctype, id) => {
   return client.getDocumentFromState(doctype, id)
 }
 
+const identity = x => x
+const portal = children => ReactDOM.createPortal(children, document.body)
```
Comment (Crash--) [inline src/ducks/recurrence/RecurrencePage.jsx:60]:
```suggestion
//see Cozy-UI's issue: https://github.com/cozy/cozy-ui/issues/1462
const identity = x => x
const portal = children => ReactDOM.createPortal(children, document.body)
```

File: webpack.config.js [hunk 3]
```diff
@@ -58,6 +59,23 @@ const modeConfig = production
 
 const config = merge(modeConfig, withTarget)
 
+const removeCSSMQPackerPlugin = config => {
```
Comment (Crash--) [inline webpack.config.js:62]: I know that you know and you'll remove those lines after an cozy-scripts upgrade, but hey adding a link to the issue and adding a todo comment can be great for external ppl.
```suggestion
//TODO remove those lines after https://github.com/cozy/create-cozy-app/pull/1326 is fixed
const removeCSSMQPackerPlugin = config => {
```

File: src/ducks/account/AccountSwitch.styl [hunk 4]
```diff
@@ -1,7 +1,5 @@
 // TODO understand why this is necessary
 // for the account switch to work on mobile
-@require 'objects/*'
```
Comment (Crash--) [inline src/ducks/account/AccountSwitch.styl:3]: can we remove also the comment above?

File: src/components/Header/__snapshots__/Header.spec.jsx.snap (hunk 1 above)
Comment (ptbrowne) [reply src/components/Header/__snapshots__/Header.spec.jsx.snap:19]: Good catch. A bad bit of inattention on my part. The reason is simple actually : as I was `find`ing the `divs` in the `setup`, two divs were returned (one child of the other), thus causing two elements in the array.

**Pull request merged**